README.md

```diff
@@ -277,86 +324,267 @@ covariance(height, weight)
 
 
 
-Remember that covariance is a metric that is hard to interpret. Run the cell below to visualize `height` and `weight` on a scatter plot! 
+So, we have a covariance of about 144.8. Recall the questions posed at the beginning:
+
+1. Is there a linear relationship between weight an height? 
+2. Does weight increase as height increases?
+3. How strong is the linear relationship between weight and height?
+
+Before looking at the answer below, try to identify: **Which (if any) questions can we answer with this covariance value?**
+
+.
+
+.
+
+.
+
+*Answer: we can answer questions 1 and 2.* 
+
+ - Because the covariance is not (close to) zero, we can say that there ***is*** a linear relationship between weight and height.
+ - Because the covariance is positive rather than negative, we can say that in general, yes, weight increases as height increases.
+
+So far, we cannot give a clear answer to question 3, because the scale of the covariance is based on the units of measurement in this data (inches and pounds, in this case). In order to answer question 3, let's calculate the correlation.
```

Comment:
Adding more explanation about why correlation means we can compare units directly:

So far, we cannot give a clear answer to question 3, because the scale of the covariance is based on the units of measurement in this data (inches and pounds, in this case).  

While the covariance can be used to figure out *in which direction* two variables have a linear relationship - does one increase as the other decreases, or vice versa - any conclusion we might draw about the *strength* of the linear relationship from the covariance might be an artifact of trying to compare these dis-similar units.  

Think of comparing the covariance between height and weight, and the covariance between height and resting heart rate.  We can't really tell if the differences between those two covariances are due to the *behavior of the data* or to the fact that *one comparison is between inches and pounds, and the other is between inches and seconds*.

However, if we had a measure that compared like units with like units, we would be able to find the strength of the linear relationship between height and weight, and make a direct comparison with the strength of the relationship between height and resting heart rate.  

And we have such a measure!  **Correlation**.  

Correlation is an umbrella term that covers a number of different calculations.  The calculation we'll use below, `Pearson's correlation coefficient` or `Pearson's r`, converts the units of each variable to "units of standard deviation" and standardizes the scale of the resulting calculation from -1 to 1.

This lets us "compare like units with like units", and compare the strengths of different linear relationships of variables against each other.

Reply:
Added partially. This is not the first introduction to the purpose of covariance, this is recapping/reviewing a concept that was introduced in the [previous lesson](https://github.com/learn-co-curriculum/dsc-correlation-covariance). So I don't think we need such an extended explanation, but I did include about half of this